rails-snow-queen/app/assets/stylesheets/application.scss

```diff
@@ -36,7 +46,6 @@ header {
 }
 
 #panel {
-  background-color: #177E89;
```

Comment:
Can remove the selector now if no styles

Reply:
done